packages/frontend/src/pages/bridge-details/BridgeDetailsPage.tsx

```diff
@@ -60,11 +63,25 @@ export const BridgeDetailsPage = () => {
         </IconButton>
       </Box>
 
-      <BridgeDetails bridge={bridge} />
+      <MemoizedBridgeDetails bridge={bridge} />
+
+      <Stack spacing={2}>
```

Comment:
Is this extra stack necessary? Can’t we use the parent stack somehow?

Reply:
I could definitely use the parent stack and its spacing. 
But from a UI/UX perspective I feel like having a smaller gap between the device list and the header/button/timer kind of groups them together and is more intuitive.